packages/financial-templates-lib/src/helpers/multicall.js

```diff
@@ -0,0 +1,23 @@
+// Provides convenience methods for interacting with deployed Multicall contract on network.
+const { getAbi } = require("@uma/core");
+const multicallAddressMap = require("@makerdao/multicall/src/addresses.json");
+const assert = require("assert");
+
```

Comment:
We could have this method automatically decode the outputs:
```js
const { getAbiDecoder } = require("@uma/common");

let abiDecoder;

function _decodeOutput(callData, returnData, web3) {
  let abiDecoder = abiDecoder || getAbiDecoder(); // Only load once because it's expensive.
  const methodAbi = abiDecoder.getMethodIDs()[callData.slice(2,10)];
  return web3.eth.abi.decodeParameters(methodAbi.outputs, returnData);
}
```

Then we could call this in `aggregateTransactionsAndCall` to auto-decode.

Thoughts?

We could also do something like this in a follow-up.

Reply:
Very nice, it worked and I refactored

Reply:
I really like this change. good implementation!